config/application.rb

```diff
@@ -40,7 +40,11 @@ class Application < Rails::Application
     config.assets.version = '1.0'
 
     config.assets.paths << "#{Rails.root}/assets/fonts"
-    config.autoload_paths += %W(#{config.root}/lib #{config.root}/lib/modules)
+    config.autoload_paths += %W(
+      #{config.root}/lib 
+      #{config.root}/lib/modules 
+      #{config.root}/app/serializers
```

Comment:
I thought serializers were loaded by default - is this not the case?

Reply:
Unfortunately not, if they aren't included in the autoloaded paths, then they are not converted to constants and any references to them in other files will result in Uninitialized Constant errors

Reply:
Strange, for me it seems to work fine if I remove the line. We also have serializers in Encore, but are not explicitly autoloading them as far as I can tell. Could it be something to do with restarting the server or perhaps having to restart terminal in order to trigger the loading? I often have to explicitly load a file or reload terminal if something in lib/modules is changed for example.

Reply:
Yeah it's weird, it does work without it. I'm going to remove it for now.  The `lib` folders though would need to stay in though as they're not part of `app` so wouldn't be eager or auto loaded.